js-intro.js

```diff
@@ -47,10 +70,23 @@ var myMessage = "Learning is fun!"
 
 // 3a. Write the code that logs each letter of the message using a for loop.
 //hint: "logs" in this case means "prints to the terminal console."  This is typically done with a console.log(x), where x is the string, function, etc that you wish to see in the terminal.
-
+const learning = (string) => {
+    let newStr = myMessage
+    for(let i=0; i < string.length; i++){
+ }
+ return newStr
+}
+console.log(learning(myMessage))
 
 // 3b. Write the code that logs each letter of the message using map.
+var myArr = [myMessage]
 
+const learn = () => { 
+    let newArr  = myArr.map(value => value = myArr.toString())
+    return newArr
+}
+
+console.log(learn(myArr))
 
```

Comment:
Here the output of your console.log is [ 'Learning is fun!' ]
To get each letter of the array:
const learn = (string) => {
    let newArr = string.split("").map(value => value)
    return newArr
}

console.log(learn(myMessage))